WNPRC_Purchasing/src/client/components/LineItemRow.tsx

```diff
@@ -55,10 +56,25 @@ export const LineItemRow: FC<LineItemProps> = memo(props => {
         onDelete(rowIndex);
     }, [onDelete, rowIndex]);
 
+    let columnCount = 8;
```

Comment:
that's the minimum number of columns that there are in the 'Specify Items' section of the purchasing request form. then column widths are calculated based on the final column count so that they always span the width of the page.